Packs/SymantecBlueCoatProxySG/ModelingRules/SymantecBlueCoatProxySG/SymantecBlueCoatProxySG.xif

```diff
@@ -8,33 +8,30 @@
 // filter out records which contain only metadata 
 filter _raw_log !~= "^#.*" 
 
-| alter // Extract all of the values into Array
-    field_list = arraymap(regextract(_raw_log, "(?:\"[^\"]+\")|[\S]+"), if("@element" ~= "^\"", replex("@element", "\"", ""), "@element"))
-
 | alter // Extract fields based on their relative location in the list
-    time_taken = arrayindex(field_list, 2), // Time taken (in milliseconds) to process the request
-    c_ip = arrayindex(field_list, 3), // IP address of the client
-    sc_status = arrayindex(field_list, 4), // Protocol status code from appliance to client
-    s_action = arrayindex(field_list, 5), // Type of action the Edge SWG (ProxySG) took to process this request (see https://techdocs.broadcom.com/us/en/symantec-security-software/web-and-network-security/edge-swg/7-4/getting-started/page-help-administration/page-help-logging/log-formats.html#concept26560_ActionF)
-    sc_bytes = arrayindex(field_list, 6), // Number of bytes sent from appliance to client
-    cs_bytes = arrayindex(field_list, 7), // Number of bytes sent from client to appliance
-    cs_method = arrayindex(field_list, 8), // Request method that is used from client to appliance
-    cs_uri_scheme = arrayindex(field_list, 9), // Scheme from the log URL.
-    cs_host = arrayindex(field_list, 10), // Hostname from the log URL
-    cs_uri_port = arrayindex(field_list, 11), // Port from the log URL.
-    cs_uri_path = arrayindex(field_list, 12), // Path from the log URL. Does not include the query.
-    cs_uri_query = arrayindex(field_list, 13), // Query from the log URL
-    cs_username = arrayindex(field_list, 14), // username of a client authenticated to the proxy
-    cs_auth_group = arrayindex(field_list, 15), // group to which an authenticated user belongs
-    s_supplier_name = arrayindex(field_list, 16), // Hostname of the upstream host
-    rs_content_type = arrayindex(field_list, 17), // Response header: Content-Type
-    cs_referer = arrayindex(field_list, 18), // Request header: Referer 
-    cs_user_agent = arrayindex(field_list, 19), // Request header: User-Agent
-    sc_filter_result = arrayindex(field_list, 20), // Content filtering result: Denied, Proxied, or Observed 
-    cs_categories = arrayindex(field_list, 21), // Content filtering category of the request URL
-    x_virus_id = arrayindex(field_list, 22), // Identifier of a virus if one was detected.
-    s_ip = arrayindex(field_list, 23), // IP address of the appliance on which the client established its connection
-    r_ip = arrayindex(field_list, 24) // IP address of the destination
+    time_taken = regexcaptureResult -> time_taken, // Time taken (in milliseconds) to process the request
```

Comment:
Lets change the generated JSON field name to **parsed_fields**
This is the name we use across all of the packs we use regexcapture in them.